controls/radchartview/features/plot-area.md

```diff
@@ -51,6 +51,42 @@ You can customize the plot area visual via the __PlotAreaStyle__ property of the
 #### __Figure 3: PlotAreaStyle example__
 ![PlotAreaStyle example](images/radchartview-features-plot-area-2.png)
 
+## Displaying stacked elements 
+
+By default the order in which the elements are defined in XAML determines which will be in front of the other when displayed. You can control which element is displayed on top by setting the elements' **ZIndex** property. RadChartView elements which have this property are [Series]({%slug radchartview-series-chartseries%}), [Annotations]({%slug radchartview-annotations-overview%}) and [Axes]({%slug radchartview-axes-axis%}).
+
+#### __[XAML] Example 3: Setting ZIndex property__
+{{region radchartview-features-plot-area-2}}
+	<telerik:RadCartesianChart x:Name="chart" Palette="Windows8">
+            <telerik:RadCartesianChart.HorizontalAxis>
+                <telerik:CategoricalAxis/>
+            </telerik:RadCartesianChart.HorizontalAxis>
+
+            <telerik:RadCartesianChart.VerticalAxis >
+                <telerik:LinearAxis />
+            </telerik:RadCartesianChart.VerticalAxis>
+
+            <telerik:RadCartesianChart.Series>
+                <telerik:BarSeries>
+                    <telerik:BarSeries.DataPoints>
+                        <telerik:CategoricalDataPoint Category="January" Value="3" />
+                        <telerik:CategoricalDataPoint Category="February" Value="5" />
+                    </telerik:BarSeries.DataPoints>
+                </telerik:BarSeries>
+
+                <telerik:LineSeries ZIndex="100">
```

Comment:
I am not sure that ZIndex=100 will lead to the visualization in the attached picture. You should be able to get this effect even if the ZIndex is not set, because the LineSeries is defined after the BarSeries. 

Also, I think the series has default ZIndex of 200 if I am not mistaken. Can you check this please? If so, can you check what is the default ZIndex of the different elements (Axis, Series and Annotations) and list them in the section.

I think it would be better to have the line series defined behind the bar series and then set its ZIndex so it is displayed on top of the bar series.